protocol/communities_messenger_test.go

```diff
@@ -2172,7 +2172,7 @@ func (s *MessengerCommunitiesSuite) TestLeaveAndRejoinCommunity() {
 			numberInactiveChats++
 		}
 	}
-	s.Require().Equal(3, numberInactiveChats)
+	s.Require().Equal(4, numberInactiveChats)
```

Comment:
I don't understand why the number of inactive chats is now higher?

Reply:
It's because of the backup, which backs up chat as well. because the community is left, 1 more chat is inactive. Did I miss something?